data_buffer_transfer/src/main/java/com/kingland/eip/Application.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2020 Kingland Systems Corporation. All Rights Reserved.
+ */
+package com.kingland.eip;
+
+import com.kingland.eip.common.Consts;
+import com.kingland.eip.common.buffer.DataBuffer;
+import com.kingland.eip.data_transfer.DataLoader;
+import com.kingland.eip.data_transfer.DataSender;
+import com.kingland.eip.datasource.MultipleDataSources;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Application {
+    public static void main(String[] args) throws Exception {
+        final int capacity = 100;
+        final int threads = 2;
+        String sourcePath = null;
+        List sourceList = new ArrayList<>();
```

Comment:
The main process should be create data source, load data and send data to the destination file